clientgui/ProjectInfoPage.cpp

```diff
@@ -643,25 +702,42 @@ void CProjectInfoPage::OnPageChanged( wxWizardExEvent& event ) {
                         pProjectInfo->m_bProjectSupportsLinux = true;
                     }
 
+                    if (strProjectPlatform.Find(_T("android")) != wxNOT_FOUND) {
+                        pProjectInfo->m_bProjectSupportsAndroid = true;
+                    }
+
                     if (strProjectPlatform.Find(_T("freebsd")) != wxNOT_FOUND) {
                         pProjectInfo->m_bProjectSupportsFreeBSD = true;
                     }
-                    
-                    if (strProjectPlatform.Find(_T("android")) != wxNOT_FOUND) {
-                        pProjectInfo->m_bProjectSupportsAndroid = true;
+
+                    if (strProjectPlatform.Find(_T("arm-unknown-linux-gnu")) != wxNOT_FOUND) {
+                        pProjectInfo->m_bProjectSupportsLinuxARM = true;
+                    }
+
+                    if (strProjectPlatform.Find(_T("arm-unknown-linux-gnueabihf")) != wxNOT_FOUND) {
+                        pProjectInfo->m_bProjectSupportsLinuxARM = true;
                     }
 
-                    if (strProjectPlatform.Find(_T("[cuda")) != wxNOT_FOUND) {
+                    if (strProjectPlanClass.Find(_T("cuda")) != wxNOT_FOUND) {
                         pProjectInfo->m_bProjectSupportsCUDA = true;
 						if (!pDoc->state.host_info.coprocs.have_nvidia()) continue;
                     }
 
-                    if (strProjectPlatform.Find(_T("[ati")) != wxNOT_FOUND) {
+                    if (strProjectPlanClass.Find(_T("nvidia")) != wxNOT_FOUND) {
+                        pProjectInfo->m_bProjectSupportsIntelGPU = true;
```

Comment:
Bug, though checking for `nvidia` makes sense to catch NVIDIA+OpenCL.

Reply:
Good catch. I'll check for that and other copy'n'paste errors in the morning.